listeners/websocket.js

```diff
@@ -0,0 +1,64 @@
+var WebSocketClient = require('websocket').client;
+var deviceInfo = require('./../data/device'),
```

Comment:
Of course not. Good catch.
Location of device config should be read from global config.